KeePassRPC/KeePassRPCService.cs

```diff
@@ -1556,7 +1588,7 @@ private LightEntry[] GetChildEntries(PwDatabase pwd, PwGroup group, bool fullDet
                     if (EntryIsInRecycleBin(pwe, pwd))
                         continue; // ignore if it's in the recycle bin
 
-                    if (string.IsNullOrEmpty(pwe.Strings.ReadSafe("URL")))
+                    if (string.IsNullOrEmpty(pwe.Strings.ReadSafe("URL")) && !everyEntry)
```

Comment:
This would be (fractionally!) faster if you put the boolean test before the string test.